html/templates/playerdisplaycombatant.html

```diff
@@ -2,7 +2,7 @@
 <div class="name">
     <span data-bind="text: Name, attr: {title: Name }"></span>
 </div>
-<div class="hp" data-bind="css: { 'disable-hover': $root.DisallowSuggestions }">
+<div class="hp" data-bind="css: { 'disable-hover': !$root.AllowSuggestions }">
```

Comment:
Player view still shows hover even if AllowSuggestions is false (seems like a KO bug)